src/style.scss

```diff
@@ -175,17 +177,153 @@
     padding: var(--auro-size-xs) 0;
   }
 
-  .details-slot {
+  .detailsSlot {
     padding: 0;
   }
 }
 
+// =-=-=-=-=-=-=-=-=-=-= shade =-=-=-=-=-=-=-=-=-=-=-=-
+%slottedHeading {
+  margin: 0;
+  font-size: var(--auro-text-heading-400-size);
+  font-weight: var(--auro-text-body-default-weight);
+}
+
 :host([shade]) {
   @extend %chromeless;
   @extend %lowProfile;
 
+  position: relative;
+  display: block;
+
+  .details {
+    position: absolute;
+    width: 100%;
+    z-index: 1;
+    background-color: var(--auro-color-background-lighter);
+    transition: all 0.2s ease-in-out;
+  }
+
+  .trigger {
+    &::slotted(h1) {
+      @extend %slottedHeading
+    }
+
+    &::slotted(h2) {
+      @extend %slottedHeading
+    }
+
+    &::slotted(h3) {
+      @extend %slottedHeading
+    }
+
+    &::slotted(h4) {
+      @extend %slottedHeading
+    }
+  }
+
   .detailsTrigger {
-    font-size: var(--auro-text-body-size-default);
+    font-size: var(--auro-text-heading-400-size);
     font-weight: var(--auro-text-body-default-weight);
+    background-color: var(--auro-color-background-lighter);
+    padding-top: var(--auro-size-sm);
+    padding-bottom: var(--auro-size-sm);
+
+    @include auro_breakpoint--md {
+      padding-left: 0;
+      padding-right: 0;
+    }
+  }
+
+  .detailsSlot,
+  .triggerContentWrapper {
+    max-width: 768px;
+    padding: 0 1rem;
+    margin: 0 auto;
+
+    @include auro_breakpoint--lg {
+      padding-left: 0;
+      padding-right: 0;
+    }
+  }
+}
+
+:host([shade][information]),
+:host([shade][warning]),
+:host([shade][error]) {
+  .details,
+  .detailsTrigger {
+    border-bottom-width: 4px;
+    border-bottom-style: solid;
+  }
+}
+
+:host([shade][information]) {
+  .detailsTrigger {
+    border-bottom-color: var(--auro-color-alert-notification-on-light);
+
+    &.expanded {
+      border-bottom-color: var(--auro-color-background-lighter);
+    }
+  }
```

Comment:
Consider a simpler one stop shop

```
.detailsTrigger {
    &:not(.expanded) {
      border-bottom-color: var(--auro-color-alert-notification-on-light);
    }
 }
```

A different route I would probably take that I think is even simpler would be to just put the border div.details and leave it there all the time combined with simply toggling the display of div.detailsSlot. This would remove a lot of these lines of SCSS.

Reply:
I am not a fan of reverse logic and prefer declarative statements. In this scenario, it is clear that the `expanded` option is used when the `expanded` selector is on the DOM. 

The ways you suggest are somewhat sudo trickery and a ready may have to decipher what is happening to understand what is happening. 

Declarative readability always wins out in areas of 'what we can do' versus 'what we should do'

Reply:
It certainly isn't wrong to be more verbose. I disagree about readability in this case. Several more lines of code just to avoid using a :not() selector is far less readable I believe and requires more mental gymnastics to figure out how all the selectors work together. But, your code, write it how you prefer. :)

Reply:
It's not that it's a preference, but consistency. Also, I need to update this to be `.isExpanded` per my own rules that I am breaking https://auro.alaskaair.com/support/css-conventions

Negative logic is acceptable in a pinch, but if there is a declarative way to write it, I gravitate to that. 

This article sums up my opinion on this https://www.obsidiansecurity.com/blog/why-to-stop-writing-negative-code/